.github/workflows/test-windows.yml

```diff
@@ -24,18 +24,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12-dev"]
-        architecture: ["x86", "x64"]
-        include:
-          # PyPy 7.3.4+ only ships 64-bit binaries for Windows
-          - python-version: "pypy3.9"
-            architecture: "x64"
-          - python-version: "pypy3.10"
-            architecture: "x64"
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12-dev", "pypy3.9", "pypy3.10"]
```

Comment:
PyPy is up to twice as slow as CPython tests:

<img width="322" alt="image" src="https://github.com/python-pillow/Pillow/assets/1324225/182e78c9-82ca-48d6-b9bc-953d4a24b8d9">

Like test.yml, let's put pypy3.x first to give it a head start:

```suggestion
        python-version: ["pypy3.10", "pypy3.9", "3.8", "3.9", "3.10", "3.11", "3.12-dev"]
```